src/client/helpers/entity.js

```diff
@@ -168,6 +168,18 @@ export function getEntityDisambiguation(entity) {
 	return null;
 }
 
+export function getEntityAlias(entity) {
+	const aliases = entity.aliasSet.aliases
+		.map(item => item.name)
+		.filter(item => item !== entity.defaultAlias.name)
```

Comment:
I think it would be more precise and safer to use the `defaultAliasId` property to filter out the default alias (or perhaps `defaultAlias.id`, I doubt it matters).
I'm thinking that you could theoretically have two different aliases with the exact same name, and that would be useful to know when you display the page (so that for example a user can remove the excess alias).

That also means that you'll need to call `.filter`before you call `.map`, since after the map you won't have access to the item.id anymore.

Reply:
Good idea--it might also be worth removing dupicates in the AliasEditor at some point?